addons/mail/static/tests/mail_test_helpers_contains.js

```diff
@@ -958,6 +958,7 @@ export async function contains(selector, options) {
 
 const stepState = {
     expectedSteps: null,
+    /** @type {import("@odoo/hoot-mock").Deferred} */
     deferred: null,
```

Comment:
This is just so there's no warning on `await assertSteps()` suggesting to remove the `await` when it's definitely necessary!